pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -107,25 +107,28 @@ func SetDomainSpec(virConn cli.Connection, vmi *v1.VirtualMachineInstance, wante
 func GetDomainSpec(status libvirt.DomainState, dom cli.VirDomain) (*api.DomainSpec, error) {
 
 	var spec *api.DomainSpec
-	inactiveSpec, err := GetDomainSpecWithFlags(dom, libvirt.DOMAIN_XML_INACTIVE)
+	// inactiveSpec, err := GetDomainSpecWithFlags(dom, libvirt.DOMAIN_XML_INACTIVE)
```

Comment:
You can remove the commented-out code.

Reply:
Ah, the remains of debuging. Will remove.